Controllers/ApplicationController.cs

```diff
@@ -144,20 +433,29 @@ public Task<string> RedeemPopup()
             // set CSRF token in browser cookie
             response.Cookies.Add(cookie);
 
+            DateTime startDate = DateTime.Now;
+            DateTime endDate = startDate.AddYears(1);
+
             return GetMerchantMember().FlatMap(mem => mem.GetFirstAlias()
-               .FlatMap(alias => mem.StoreTokenRequest(
-                   // Create a token request to be stored
-                   TokenRequest.TransferTokenRequestBuilder(amount, currency)
-                       .SetDescription(description)
-                       .AddDestination(destination)
-                       .SetRefId(refId)
-                       .SetToAlias(alias)
-                       .SetToMemberId(mem.MemberId())
-                       .SetRedirectUrl(callbackUrl)
-                       .SetCsrfToken(csrfToken)
-                       .Build()))
-               // generate the Token request URL to redirect to
-               .FlatMap(requestId => tokenClient.GenerateTokenRequestUrl(requestId)));
+              .FlatMap(alias => mem.StoreTokenRequest(
+                  // Create a token request to be stored
+                  TokenRequest.StandingOrderRequestBuilder(
+                            amount,
+                            currency,
+                            "MNTH",
+                            startDate.ToString("yyyy-MM-dd"),
+                            endDate.ToString("yyyy-MM-dd"),
+                            new List<TransferDestination> { destination })
+                        .SetDescription(description)
+                        .AddDestination(destination)
```

Comment:
The TokenRequest already sets the destination with StandingOrderRequestBuilder, do we need to addDestination again?

Reply:
Fixed. Done